docs/resources/ledger.md

```diff
@@ -31,21 +31,18 @@ To see the `Cosmos` application when you search for it, you might need to activa
 
 ![Devmode](../images/ledger-tuto-dev-mode.png)
 
-## Lunie.io + Ledger Nano  
```

Comment:
I believe there's still a reference to Lunie at the top of the docs.

Reply:
Whoopsie, quite a bit of them haha. Fixed now!